cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/FoldBlocks.hs

```diff
@@ -32,7 +31,7 @@ import           Testnet.Runtime
 newtype FoldBlocksException = FoldBlocksException Api.FoldBlocksError
 instance Exception FoldBlocksException
 instance Show FoldBlocksException where
-  show (FoldBlocksException a) = TS.unpack $ Api.renderFoldBlocksError a
+  show (FoldBlocksException a) = show a
```

Comment:
actually this should be `docToString $ prettyError a`

Reply:
Closing, because this change was part of working with a custom cardano-api version, and is not there anymore since rebasing on 8.8-pre.